exercises/express-static/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+
+const path = require('path');
+
+const app = express();
+
+path.join('./public', __dirname);
```

Comment:
this is just discarding `path.join()`'s return value. 

![image](https://user-images.githubusercontent.com/7432943/125339431-23828f80-e306-11eb-99f1-94edbb64fe5f.png)